### DISCUSSION TITLE
Add the "stop avatars from scrolling" option back to tweaks?

### DISCUSSION THREAD

**Discussion by ghost** [AprilSylph/XKit-Rewritten#239] [Suggestion Box]
I know it's a small and mostly useless feature, but I really appreciated having it (something about the regular avatars is extremely distracting while I'm scrolling, I wish I could explain this but I have no idea) and the only other extension I've found with this function doesn't seem to work anymore. 

I completely understand something like this being very low on the priority list or not worth the effort, but I didn't see it anywhere in the Projects tab and wanted to mention it.

**Comment by AprilSylph** [comment]
This was implemented in 10e592f1cf7a063f184910f3c0dbb0e386d60809 and released in v0.8.4!
